db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+require './spec/helpers/factory_helpers'
```

Comment:
i think you may have merged this file backwards. we want to delete this line and create the hearings for different weeks below.

Reply:
yeah there's no need for any changes to this file in this PR, will remove it